java/arcs/core/host/ArcHostContextParticle.kt

```diff
@@ -64,7 +69,7 @@ class ArcHostContextParticle(
             val storedConnections = handles.handleConnections.fetchAll()
 
             val particles = context.particles.map {
-                ArcHostContextParticle_Particles(
+                ParticleSchema(
```

Comment:
I still don't love that this is named ParticleSchema, and yet is actually an entity. Here's a few other ideas: SerializedParticle, ArcHostContextParticle, ContextParticle, ParticleInContext. 

For now it would be ok to just add a TODO in the .arcs file to rename these "FooSchema" schemas.